src/styles/main.scss

```
@@ -1,3 +1,63 @@
+@import "variables.scss";
+@import "normalize.scss";
+
 body {
   margin: 0;
 }
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  padding: $calendar-padding;
+  max-width: $calendar-max-width;
+
+  @each $week-day, $margin in $start-day {
+    &--start-day-#{$week-day} {
+      .calendar__day:first-child {
+        margin-left: $margin;
+      }
+    }
+  }
+
+  @for $month-days from 28 to 31 {
+    &--calendar-length-#{$month-days} {
+      .calendar__day:nth-child(n + #{$month-days + 1}) {
+        display: none;
+      }
+    }
+  }
+
+  &__day {
+    display: flex;
+    width: $cell-size;
+    height: $cell-size;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+
+    background-color: $bcgColor;
+    border: 1px solid $borderColor;
+    transition: transform 0.5s linear, background-color 0.5s linear;
```

create a variable for transition duration, bc you will reuse this value in all your page
```suggestion
    transition: transform $transformDuration linear, background-color $transformDuration linear;
```